src/chart/sankey/SankeyView.js

```diff
@@ -132,6 +154,7 @@ export default echarts.extendChartView({
         var width = layoutInfo.width;
         // view height
         var height = layoutInfo.height;
+        var mode = seriesModel.option.mode || 'normal';
```

Comment:
It's suggested to use `seriesModel.get('mode')` instead of `seriesModel.option.mode`.
```suggestion
        var mode = seriesModel.get('mode') || 'normal';
```